website/docs/concepts/function-components/properties.mdx

```diff
@@ -158,26 +158,32 @@ use yew::{function_component, html, Html, Properties};
 
 #[derive(Properties, PartialEq)]
 pub struct Props {
+    #[prop_or_default]
+    pub is_loading: bool,
     // highlight-start
-    #[prop_or("Bob".to_string())]
+    #[prop_or(AttrValue::Static("Bob"))]
     // highlight-end
-    pub name: String,
+    pub name: AttrValue,
 }
 
 #[function_component]
-fn HelloWorld(props: &Props) -> Html {
-    html! {<>{"Hello world"}{props.name.clone()}</>}
+fn Hello(&Props { is_loading, ref name }: &Props) -> Html {
```

Comment:
It's certainly handy to know, but not to always use, since writing the name of the props type twice can get cumbersome really fast. If only Rust had something like anonymous struct patterns...

Reply:
YESSS I know..... well at least autoprops makes it easier in both ways (Prop struct creation + destructuring)